package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hof",
   "description": "A bootstrap for HOF projects",
-  "version": "20.0.1",
```

Comment:
I don't believe you need to up the version number, git actions will do it for you when it is merged

Reply:
thanks Sulthan - I've rebased following Alex's changes so this change should be removed now